DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/RemoveProcessor.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.Anonymizer.Core.Model;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors
+{
+    public class RemoveProcessor : IAnonymizerProcessor
```

Comment:
What's the difference between redact and remove in dicom? We can add some comments before the processors or method definitions.